README.md

```diff
@@ -325,6 +325,22 @@ dokku scheduler-kubernetes:service-annotations-set node-js-sample pod.kubernetes
 
 Currently, they are applied to the `web` process, which is the only process for which a Kubernetes Service is created.
 
+#### Ingress Annotations
+
+These can be managed by the `:ingress-annotations-set` command.
+
+```shell
+# command structure
+dokku scheduler-kubernetes:ingress-annotations-set $APP name value
+
+# set example
+dokku scheduler-kubernetes:ingress-annotations-set node-js-sample nginx.ingress.kubernetes.io/affinity cookie
```

Comment:
So like, we currently use one ingress for all apps in the same namespace. How does this work given that annotations are being set per app? Wouldn't apps conflicts?

Reply:
> So like, we currently use one ingress for all apps in the same namespace. How does this work given that annotations are being set per app? Wouldn't apps conflicts?

Really good point, will refactoring this to namespace instead of app be fine then?

Reply:
oooo good call.